### PR TITLE
api-gateway-dev.yaml file updated with master credentials,clientId,authServerURL

### PR DESCRIPTION
**Pull request by denduluriramasaimohanakhil**

### CODE REVIEW

File: tp-api-gateway-dev.yaml [hunk 1]
```diff
@@ -1,6 +1,16 @@
 keycloak:
   issuer-uri: ${KEYCLOAK_URL_AUTH:https://keycloak-tsplatform.techsophy.com}/auth/realms/
   realm.name: ${KEYCLOAK_REALM_NAME:techsophy-platform}
+  client.id:  ${CAMUNDA_IDENTITY_SERVICE:camunda-identity-service}
+  auth-server-url: ${KEYCLOAK_URL_AUTH:https://keycloak-tsplatform.techsophy.com}/auth
+  master:
+      username: ${MASTER_USERNAME:venkataramana}
```
Comment (VaibhavTechsophy) [inline tp-api-gateway-dev.yaml:7]: no default user should be configured in the realm, does this user has access to all the realms?

File: tp-api-gateway-dev.yaml [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+      password: ${MASTER_PASSWORD:Tech@4567}
```
Comment (VaibhavTechsophy) [inline tp-api-gateway-dev.yaml:8]: no password should be committed either

**Review by VaibhavTechsophy — CHANGES_REQUESTED** [review]
Please create appropriate user and environment variables created, if the merge happens it should not break the code.

File: tp-api-gateway-dev.yaml (hunk 1 above)
Comment (denduluriramasaimohanakhil) [reply tp-api-gateway-dev.yaml:7]: yes vaibhav removing default user

File: tp-api-gateway-dev.yaml (hunk 2 above)
Comment (denduluriramasaimohanakhil) [reply tp-api-gateway-dev.yaml:8]: yes vaibhav removing default password

**Pull request closed without merging**